tachyon/math/polynomials/multivariate/multilinear_extension_ops.h

```diff
@@ -175,13 +180,18 @@ class MultilinearExtensionOp<MultilinearDenseEvaluations<F, MaxDegree>> {
   static MultilinearExtension<D>& DivInPlace(
       MultilinearExtension<D>& self, const MultilinearExtension<D>& other) {
     std::vector<F>& l_evaluations = self.evaluations_.evaluations_;
+    const std::vector<F>& r_evaluations = other.evaluations_.evaluations_;
     if (l_evaluations.empty()) {
       // 0 / g(x)
       return self;
     }
-    // TODO(chokobole): Check division by zero polynomial.
-    // f(x) / 0 skips this for loop.
-    const std::vector<F>& r_evaluations = other.evaluations_.evaluations_;
+    if (r_evaluations.empty()) {
+      // f(x) / 0
+      // TODO(chokobole): It should return std::nullopt.
+      // See https://github.com/kroma-network/tachyon/issues/76.
+      return self;
+    }
```

Comment:
I think it's better to check `r_evaluation.empty()` before `l_evaluation.empty()`.